src/ophyd_async/epics/core/_aioca.py

```diff
@@ -40,6 +41,10 @@
 logger = logging.getLogger("ophyd_async")
 
 
+def _default_all_updates() -> bool:
+    return os.environ.get("OPHYD_ASYNC_EPICS_CA_KEEP_ALL_UPDATES", "True") == "True"
```

Comment:
I have no strong feelings about whether the environment variable should be ``DROP_UPDATES` or `KEEP_UPDATES`